cli/attestation.go

```diff
@@ -3,22 +3,59 @@
 package cli
 
 import (
+	"fmt"
 	"log"
 	"os"
 
+	"github.com/google/go-sev-guest/abi"
+	"github.com/google/go-sev-guest/validate"
+	"github.com/google/go-sev-guest/verify"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
 const attestationFilePath = "attestation.txt"
 
 func (cli *CLI) NewAttestationCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "attestation",
-		Short: "Retrieve attestation information",
+		Use:   "attestation [command]",
+		Short: "Get and validate attestations",
 		Run: func(cmd *cobra.Command, args []string) {
-			log.Println("Checking attestation")
+			fmt.Printf("Get and validate attestations\n\n")
```

Comment:
Can we use log.Println to be consistent?

Reply:
Use it in combination with fmt.Sprintf

Reply:
these are not log statements